frontend/src/routes/Governance/components/CodeBlock.tsx

```diff
@@ -0,0 +1,19 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { ReactNode, useEffect } from 'react'
+import hljs from 'highlight.js'
+import diff from 'highlight.js/lib/languages/diff'
+import './github.min.css'
+
+export const CodeBlock = ({ children }: { children: ReactNode }) => {
+  useEffect(() => {
+    hljs.registerLanguage('diff', diff)
+    hljs.highlightAll()
+  }, [])
+  return (
+    <div>
+      <pre>
+        <code className="language-diff">{children}</code>
+      </pre>
+    </div>
+  )
```

Comment:
I prefer just highlighting the block in the component like this.
```suggestion
  const codeRef = useRef<ElementRef<'code'>>(null)

  useEffect(() => {
    if (codeRef?.current) {
      hljs.registerLanguage('diff', diff)
      hljs.highlightBlock(codeRef.current)
    }
  }, [])

  return (
    <div>
      <pre>
        <code ref={codeRef} className="language-diff">
          {children}
        </code>
      </pre>
    </div>
  )
  )
```